charts/cmc-claim-store/values.preview.template.yaml

```diff
@@ -35,18 +34,16 @@ java:
     IDAM_API_URL: https://idam-api.aat.platform.hmcts.net
     PDF_SERVICE_URL: http://cmc-pdf-service-aat.service.core-compute-aat.internal
     DOCUMENT_MANAGEMENT_URL: http://dm-store-aat.service.core-compute-aat.internal
-    PAY_URL: false
-    FEES_URL: false
     DOC_ASSEMBLY_URL: false
     FRONTEND_BASE_URL: https://www-${SERVICE_FQDN}
     RESPOND_TO_CLAIM_URL: https://www-${SERVICE_FQDN}/first-contact/start
-    CLAIM_STORE_DB_HOST: ${SERVICE_NAME}-postgres
-    CLAIM_STORE_DB_USERNAME: hmcts
-    CLAIM_STORE_DB_PASSWORD: hmcts
     CMC_DB_HOST: ${SERVICE_NAME}-postgres
     CMC_DB_USERNAME: hmcts
     CMC_DB_PASSWORD: hmcts
     CORE_CASE_DATA_API_URL: http://${SERVICE_NAME}-data-store-api
+    FEATURE_TOGGLES_SAVE_CLAIM_STATE_ENABLED: true
+    FEATURE_TOGGLES_ASYNC_EVENT_OPERATIONS_ENABLED: true
```

Comment:
I thought this was mean to be removed by cleanup @dharmendrak was working on

Reply:
done